packages/makeen-user/src/libs/passport.js

```diff
@@ -65,11 +61,10 @@ export default ({ jwtSecret }) => {
     cb(null, user._id.toString());
   });
 
-  passport.deserializeUser((req, id, cb) => {
-    req.app.modules
-      .get('makeen.user.UserRepository')
-      .findById(objectId(id))
-      .then(user => cb(null, user), cb);
+  passport.deserializeUser(async (req, id, cb) => {
+    const { UserRepository } = await req.app.modules.get('makeen.user');
```

Comment:
no need for that - modules.get('some-module-name') is synchronous and it returns a map of that module's exports